Code/Framework/AzToolsFramework/AzToolsFramework/Metadata/MetadataManager.cpp

```diff
@@ -28,7 +28,7 @@ namespace AzToolsFramework
 
     bool MetadataManager::GetValue(AZ::IO::PathView file, AZStd::string_view key, void* outValue, AZ::Uuid typeId)
     {
-        rapidjson_ly::Document value;
+        rapidjson::Document value;
```

Comment:
ditto here - does your change above not revert to using the `rapidjson_ly` namespace?

Reply:
nvm I see the alias `namespace rapidjson = rapidjson_ly;`

Reply:
It aliases rapidjson to rapidjson_ly actually
```
namespace rapidjson = rapidjson_ly;
```

So in our code, either works.